src/Field/BasicField.php

```diff
@@ -115,6 +115,12 @@ public function fetchFieldKey($fieldName)
      */
     public function fetchFieldType($fieldKey)
     {
+        // Check if this a clone field. If it is, return the field type of its clone.
+        $fieldKeyIds = explode('_', $fieldKey);
+        if (count($fieldKeyIds) == 4) {
```

Comment:
I think a regex is better here. Something like `/field_([A-Za-z0-9_-]+)_foo_bar/`. Whenever you can please take a look on this. Don't worry about that. Thanks for contributing. JG